app/views/posts/show.html.erb

```diff
@@ -1,3 +1,20 @@
-<h1>Single Post</h1>
-<p><%= @placeholder_text %></p>
-<p>Find me in app/views/posts/show.html.erb</p>
+<div class="card text-bg-light mb-3" style="max-width: 60%; padding: 1rem;">
+  <div class="comm-like-container" style="display: flex; gap: 24px">
+    <h3>Post <%= @post.id %> by <%= @post.author.name %></h3>
+    <div style="display: flex; gap: 12px;">
+      <p>Number of Comments: <%= @post.comments_counter %></p>
+      <p>Number of Likes: <%= @post.likes_counter %></p>
+    </div>
+  </div>
+  <p><%= @post.text %></p>
+</div>
+
+<div class="card text-bg-light mb-3" style="max-width: 60%; padding: 1rem;">
+  <% if @comments.present? %>
+    <% @comments.each do |comment| %>
+      <%= render partial: 'comments/comment', locals: { comment: comment } %>
+    <% end %>
+  <% else %>
+    <p>No comments yet.</p>
+  <% end %>
+</div>
```

Comment:
![image](https://github.com/richie1988/blog-app/assets/93039359/7a2e485a-3c86-49ea-a5d8-e5c4ce0ae33b)
![image](https://github.com/richie1988/blog-app/assets/93039359/c50109db-40a1-43de-a76e-634ed3a50e7c)

- [ ] It would be nice if you made your post-show page show all the comments because as you can see I have comments for this post but they are not being shown on this page :fearful: Kindly fix this issue to make your app work as expected :+1: .